docs/develop/amd-vgpu.md

```
@@ -0,0 +1,119 @@
+```
+
+- `devmem`  = total device memory in MB (e.g. 196608 for MI300X).
+- `devcore` = total CU count (e.g. 304 for MI300X). 
+- `id`      = device identifier.
+
+
+### 4.2 Pod allocation (scheduler -> pod annotations -> Allocate)
+
+Allocation result is written under the **AMD-specific** key (each vendor has its own):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)